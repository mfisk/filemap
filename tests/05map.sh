#!/bin/sh

fm -vvv map -i '/test/*' wc | sed -e 's,/.*$,,' -e 's,  *, ,g' | sort
fm ls /test/*/*
