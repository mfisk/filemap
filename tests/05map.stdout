 3 9 18 
 5 15 30 
/test/f1.d/wc/.status  /test/f2.d/wc/.status
/test/f1.d/wc/stdout   /test/f2.d/wc/stdout
